helm_deploy/hmpps-template-typescript/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v2
```

Comment:
delete the /hmpps-template-typescript directory

Reply:
done